src/main/java/org/datadog/jmxfetch/util/LogLevel.java

```diff
@@ -0,0 +1,114 @@
+package org.datadog.jmxfetch.util;
+
+import java.util.logging.Level;
+
+/**
+ * LogLevel used for internal logging to match Datadog Agent levels.
+ * Comparison table with java.util.logging:
+ * <pre>
+ *  JUL     |  JMXFetch LogLevel
+ * ----------------------------
+ *  ALL     |  ALL
+ *  SEVERE  |  ERROR
+ *  WARNING |  WARN
+ *  INFO    |  INFO
+ *  CONFIG  |  DEBUG
+ *  FINE    |  DEBUG
+ *  FINER   |  TRACE
+ *  FINEST  |  TRACE
+ *  OFF     |  OFF
+ * </pre>
+ */
+public enum LogLevel {
+    ALL(0, "ALL"),
+    ERROR(1, "ERROR"),
+    WARN(2, "WARN"),
+    INFO(3, "INFO"),
+    DEBUG(4, "DEBUG"),
+    TRACE(5, "TRACE"),
+    OFF(6, "OFF");
+
+    private int level;
+    private String label;
+    private LogLevel(int level, String label) {
+        this.level = level;
+        this.label = label;
+    }
+
+    // --
+
+    /** contains returns if the given log level is contained.
+     * If this level is `OFF`, `contains` returns `true` only if `other` is `OFF`
+     * as well.
+     */
+    public boolean contains(LogLevel other) {
+        if (this == OFF) {
+            return this.level == other.level;
```

Comment:
I'm guessing more appropriate version of this line should be:
```suggestion
            return other.level == OFF;
```